src/components/Treeview/index.js

```diff
@@ -20,7 +20,7 @@ export const Treeview = ({
 	...otherProps
 }) => {
 	const [viewState, updateViewState] = useViewState(name);
-	const nodeState = { ...defaultNodeState, ...viewState.nodeState };
+	const nodeState = { ...viewState.nodeState, ...defaultNodeState };
```

Comment:
This is the opposite of what makes sense for the component. Viewstate is intended to override the default state, by design, so that user modifications are changes to it.

This is not a change that can be accepted into the library, as a result.